src/components/ToggleTodo/index.jsx

```diff
@@ -0,0 +1,22 @@
+import React, { PropTypes } from 'react'
+import CSSModules from 'react-css-modules';
+
+import styles from './styles.css';
+
+const ToggleTodo = ({onClick, actionId, phoneNumber, todo}) => {
+  return (
+    <input type="checkbox"
+           styleName='toggle-todo'
+           onChange={function() {onClick(actionId, phoneNumber)}}
+           checked={todo && todo.completed} />
```

Comment:
React yells at me in the console after I click a checkbox for the first time: "Warning: ToggleTodo is changing an uncontrolled input of type checkbox to be controlled. Input elements should not switch from uncontrolled to controlled (or vice versa). Decide between using a controlled or uncontrolled input element for the lifetime of the component."

I thought that putting the `checked` and `onChange` here would turn this into a controlled input from the start, but that doesn't seem to have done it.  :-/